.lintr

```diff
@@ -2,6 +2,8 @@ linters: all_linters(
     packages = c("lintr", "etdev"),
     pipe_consistency_linter(pipe = "%>%"),
     object_name_linter = NULL,
+    return_linter = NULL,
```

Comment:
```suggestion
    return_linter("explicit"),
```

I think this linter is still helpful to ensure consistency across the codebase. Rather than disabling it entirely, I would set it to you preferred value.